src/drivers/mailbox/bcm2835_mailbox.c

```diff
@@ -14,56 +14,56 @@
 #include <hal/mem_barriers.h>
 
 static struct bcm2835_mailbox_regs volatile *const mailbox_regs =
-		(struct bcm2835_mailbox_regs volatile *) BCM2835_MAILBOX_BASE;
```

Comment:
Did you just change tabs to spaces?
It is better if formatting is separated from logic changes!